src/commands/namespace/info.ts

```diff
@@ -41,34 +41,33 @@ export class CommandOptions extends ProfileOptions {
 
 export class NamespaceInfoTable {
     private readonly table: HorizontalTable;
-    constructor(public readonly namespace: Namespace) {
+    constructor(public readonly namespaceInfo: NamespaceInfo) {
         this.table = new Table({
             style: {head: ['cyan']},
             head: ['Property', 'Value'],
         }) as HorizontalTable;
         this.table.push(
-            ['Id', namespace.id.toHex()],
-            ['Name', namespace.name],
-            ['Registration Type', namespace.isRoot() ? 'Root Namespace' : 'Sub Namespace'],
-            ['Owner', namespace.owner.address.pretty()],
-            ['Start Height',  namespace.startHeight.compact()],
-            ['End Height', namespace.endHeight.compact()],
+            ['Id', namespaceInfo.id.toHex()],
+            ['Registration Type', namespaceInfo.isRoot() ? 'Root Namespace' : 'Sub Namespace'],
+            ['Owner', namespaceInfo.owner.address.pretty()],
+            ['Start Height',  namespaceInfo.startHeight.compact()],
+            ['End Height', namespaceInfo.endHeight.compact()],
         );
-        if (namespace.isSubnamespace()) {
+        if (namespaceInfo.isSubnamespace()) {
             this.table.push(
-                ['Parent Id', namespace.parentNamespaceId().toHex()],
+                ['Parent Id', namespaceInfo.parentNamespaceId().toHex()],
             );
         }
-        if (namespace.hasAlias()) {
-            if (namespace.alias.address) {
+        if (namespaceInfo.hasAlias()) {
+            if (namespaceInfo.alias.address) {
                 this.table.push(
                     ['Alias Type', 'Address'],
-                    ['Alias Address', namespace.alias.address.pretty()],
+                    ['Alias Address', namespaceInfo.alias.address.pretty()],
                 );
-            } else if (namespace.alias.mosaicId) {
+            } else if (namespaceInfo.alias.mosaicId) {
                 this.table.push(
                     ['Alias Type', 'MosaicId'],
-                    ['Alias MosaicId', namespace.alias.mosaicId.toHex()],
+                    ['Alias MosaicId', namespaceInfo.alias.mosaicId.toString()],
```

Comment:
this is going to fail in v0.14.0 now, as Alias.MosaicId is not instance of `MosaicId`. Should change back to toHex()

Reply:
Thanks! Will change it when updating to 0.14